Automattic/wp-calypso
# Updates force-push workflow to recommended —force-with-lease

Update to the dev docs to recommend using `--force-with-lease` over just `--force` as per `p8F0Ux-1np-p2` and https://developer.atlassian.com/blog/2015/04/force-with-lease/.

### Changes proposed in this Pull Request

* Update to DevDocs to recommend using `--force-with-lease` 

#### Testing instructions
* http://calypso.localhost:3000/devdocs/docs/git-workflow.md
* Go to the [`git-workflow` section of the dev docs](http://calypso.localhost:3000/devdocs/docs/git-workflow.md) at and read the last section on updating your PR following a `rebase`. 
* Check that it makes sense and conveys what is discussed in `p8F0Ux-1np-p2`
* Check that the spelling and rendering of markdown are correct
